### DISCUSSION TITLE
Request to Review Project Proposal

### DISCUSSION THREAD

**Discussion by yaseen2** [wagtail/gsoc#56] [General]
Is there any mentor to review my project proposal? I'll give them access to my google docs proposal.

**Comment by thibaudcolas** [comment]
Hi @yaseen2, please email me your proposal at the address listed on https://github.com/thibaudcolas. Make sure the Google Docs sharing settings are set to "Anyone with the link", with "Commenter" permissions so I can share it with other mentors.

**Reply by yaseen2** [reply]
Done ✅
